samples/net/sockets/sntp_client/prj.conf

```diff
@@ -26,3 +31,8 @@ CONFIG_LOG=y
 
 # SNTP
 CONFIG_SNTP=y
+
+#CONFIG_NET_SOCKETS_LOG_LEVEL_DBG=y
+#CONFIG_SNTP_LOG_LEVEL_DBG=y
```

Comment:
Either we enable the debug logging or leave it out completely.

Reply:
> Either we enable the debug logging or leave it out completely.

Why so strict?

Reply:
I see no benefit in leaving the commented out configs in there. Shouldn't a sample be as minimal as possible?

Reply:
> Shouldn't a sample be as minimal as possible?

I don't think that "as minimal as possible" is a requirement for samples. They should be simple and easy to understand, yes. They also should be friendly to absolute newcomers (that's how people will get acquainted with Zephyr). The samples will misbehave, and users should have means to debug them at their fingertips (like, uncomment things, instead of reading 200+ pages manuals figuring out how to enable debug logging).

Reply:
That is a valid point. But other samples do not have these config options commented out in them, ready to be activated for debugging. Why make *this* sample an exception now?

Reply:
> But other samples do not have

Some do.

> Why make this sample an exception now?

Because its author proposes this PR like that, and per above, we shouldn't discourage such practice.

Reply:
I do consider those as leftovers and they should not be in the config. What can  happen here?
- Most likely those were set to debug a certain issue, but they are not the right way or to debug and might be enabling debug just for some areas. 
- We might change the kconfig name and the sample would still have the old one which is not detected by CI because they are commented out
- It becomes a norm. New users would think this is required

How to debug the application should be rather explained in the documentation.